### ISSUE TITLE
test-message-patterns warns about duplicate promise resolution, indicates a real problem

### ISSUE THREAD

**Issue by warner** [Agoric/agoric-sdk#2393]
In current trunk (43344add608fbc78aa4428f56e2bdde590eb666c), three `test-message-pattern.js` cases emit a warning about a duplicate promise resolution:

```
doProcess: vat[v5][o+10].receive dispatch failed: p+22 was already resolved
doProcess: vat[v5][o+10].receive dispatch failed: p+20 was already resolved
doProcess: vat[v5][o+10].receive dispatch failed: p+20 was already resolved
```

This shouldn't be happening, and it indicates misbehavior by the kernel (sending a duplicate resolution), or more likely liveslots or the comms vat (getting confused about the state of a promise. It isn't causing a test failure, because the kernel tolerates the initial turn of a dispatch returning a Promise that rejects (after all the kernel shouldn't die just because one vat was briefly grumpy), but it probably should flunk the test. I'm not sure how we could escalate that kernel warning into something that AVA would see, though.

I'm not sure which specific patterns are signalling the error, so the first step is to either run each message pattern separately to track down the three cases, or find a `--verbose` flag on AVA to run the tests sequentially and print their names even though everything is passing.

**Comment by warner** [comment]
I suspect this may be related to #2363 , or at least fixing that may make this one more obvious.

**Comment by FUDCo** [comment]
closed by #2516
